src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/AbiphoneFavourites.java

```diff
@@ -204,10 +205,15 @@ public boolean onRenderStack(ItemStack stack) {
 
 	@SubscribeEvent
 	public void onDrawBackground(GuiContainerBackgroundDrawnEvent event) {
-		if (isWrongInventory()) return;
+		if (!NotEnoughUpdates.INSTANCE.hasSkyblockScoreboard()
+			|| !NotEnoughUpdates.INSTANCE.config.misc.abiphoneFavourites
+			|| Utils.getOpenChestName().equals("Abiphone Shop")
+			|| !Utils.getOpenChestName().startsWith("Abiphone ")) return;
 
 		GuiContainer container = event.getContainer();
 
+		checkShowSlot = container.inventorySlots.getSlot(13).getStack();
```

Comment:
If you are going to change something anyway, you could also split up `13` into `1*9 + 4` to more easily show which row/column you target.